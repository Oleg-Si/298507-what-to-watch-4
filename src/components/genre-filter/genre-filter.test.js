import React from 'react';
import renderer from 'react-test-renderer';
import GenreFilter from './genre-filter.jsx';

const genre = new Set(
    `Dramas`,
    `Horror`,
    `Comedies`,
    `Romance`,
    `Crime`,
    `Sci-Fi`,
    `Documentary`,
    `Thrillers`,
    `Kids & Family`
);

it(`Проверяет снепшот компонента GenreFilter`, () => {
  const tree = renderer.create(
      <GenreFilter
        genre={genre}
        activeGenre={`All genres`}
        onGenreCilck={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
