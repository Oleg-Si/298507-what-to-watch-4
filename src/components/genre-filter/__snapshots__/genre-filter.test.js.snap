// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверяет снепшот компонента GenreFilter 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      D
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      r
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      a
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      m
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      s
    </a>
  </li>
</ul>
`;
